README.md

```diff
@@ -41,6 +41,79 @@ SDC services running in
 
     open https://localhost:4443
 
+# development zone in CoaL w/ sdc-admin:master
+
+1) ssh to head node
+```bash
+ssh root@10.88.88.200
+```
+
+2) get headnode uuid:
+```bash
+$ sdc-server list
+headnode             564db874-1cca-8c84-7bfd-2602014520f9     7.0     true   running     4095  10.99.99.7
+```
+
+3) get adminui service uuid:
+```bash
+$ sapiadm showapp sdc | grep adminui
+adminui    be5e7c5d-0906-4cf5-9e87-f8ec2f85d919  2
+```
+
+4) update adminui service, add parameter server_uuid:
```

Comment:
We should be explaining *why* we are doing something. 

Under what circumstances is this step necessary? Always, otherwise `sapiadm provision` will fail?